### PR TITLE
Fixes #174

### LINKED ISSUE #174: ng-model-options updateOn blur not working
When `ng-model-options` is not specified, the default behaviour appears to be that the model is updated immediately on blur. When we add ng-model-options to increase the default debounce duration, like so:

`ng-model-options="{ updateOn: 'default blur', debounce: { 'default': 3000, 'blur': 0 }}"`

the model is not updated on blur anymore. It only is updated after three seconds when the default updateOn fires. 

See the following Plunker: https://embed.plnkr.co/XYk38kzcLu4sV1lekEKK/

1. Click on the Froala editor.
2. Type some text.
3. Click outside of the Froala editor.
See that the model has not been updated. I expected the model to be updated at this point.
4. Wait for three seconds.
See that the model is updated now.

Tested in Chrome and Firefox.

### PR DESCRIPTION
**Pull request by dheerajaccolite**

### CODE REVIEW

File: src/angular-froala.js [hunk 1]
```diff
@@ -30,8 +30,12 @@
         require: 'ngModel',
         scope: scope,
         link: function(scope, element, attrs, ngModel) {
-          if (jQuery) element = jQuery(element);
-
+          if (jQuery){ element = jQuery(element); 
+            // Create a blur event to update the data of ngModel
+            element.on('blur', function () {
+              ngModel.$setViewValue(element.innerHTML.text())
```
Comment (stefanneculai) [inline src/angular-froala.js]: We should be updating with the value from `html.get` method of the editor.

File: demo/app.js [hunk 2]
```diff
@@ -18,13 +18,15 @@
 				}
 			}
 		};
-
+		$scope.text="temp";
```
Comment (stefanneculai) [inline demo/app.js]: Why are these 2 lines for?

Comment (shashankaccolite) [reply demo/app.js]: Sorry, I added those lines for the testing purpose, those lines are not needed.

Comment (stefanneculai) [reply demo/app.js]: Please update accordingly. Thanks.

**Pull request merged**